samples/wifi/scan/src/main.c

```diff
@@ -43,6 +44,39 @@ LOG_MODULE_REGISTER(scan, CONFIG_LOG_DEFAULT_LEVEL);
 
 static uint32_t scan_result;
 
+const struct wifi_scan_params tests[] = {
+	{
+	.scan_type = WIFI_SCAN_TYPE_ACTIVE,
+	.dwell_time_active = CONFIG_WIFI_MGMT_SCAN_DWELL_TIME_ACTIVE
+	},
+#if defined(CONFIG_WIFI_MGMT_FORCED_PASSIVE_SCAN) || defined(CONFIG_WIFI_SCAN_TYPE_PASSIVE)
```

Comment:
Do we still support FORCED_PASSIVE_SCAN ? Passive scan type is good enough right?